packages/trader/src/Modules/Reports/Components/filter-component.jsx

```diff
@@ -29,10 +29,10 @@ const TransactionFilter = ({ action_type, handleFilterChange }) => {
             text: localize('Withdrawal'),
             value: 'withdrawal',
         },
-        {
-            text: localize('Transfer'),
-            value: 'transfer',
-        },
+        // {
```

Comment:
You can just remove it completely if it's not needed

Reply:
We need it later for when backend release their card. Honestly not sure if it's better to remove it or comment it. @negarn any thoughts?

Reply:
IMO, we have it in git, having it in comments just doesn't help. :) we can always look back and bring it back.

Reply:
@reza-fs @akmal-binary we have to bring it back in 2-3 days as FE needs to go live first then BE can go live then FE should go live again, so I prefer to just have it commented. up to you if you feel that strongly about it.

Reply:
I think it doesn't really matter, since we have to bring it back in 2 days, we can leave it as a comment here to save few minutes later yeah

Reply:
Seems like there are 2 different schools of thought regarding comments 😅 
But yeah, agree with Akmal in this case